### DISCUSSION TITLE
The problem of overall translation of the PNM

### DISCUSSION THREAD

**Discussion by likeduck** [PMEAL/OpenPNM#2706] [Q&A]
Dear prof，

I have recently been studying multi-scale pore network models. This figure is what I expected to be.
![Multiscale](https://user-images.githubusercontent.com/117707506/223057077-203fe420-f0d3-413c-83ba-7914232e55e2.png)
I import one PNM and creat a cubic module using openpnm, the figure is:
![Figure_1](https://user-images.githubusercontent.com/117707506/223057397-bbd0b4bd-22de-43e2-8377-ae76b7297a38.png)
The little cubic model is in the lower left corner and I want to move it to the center of a large model, but I don 't know how to do it.

Could you please give me some idea?
Thanks!

**Comment by jgostick** [comment]
You just add the desired offset to the `pn['pore.coords']` array, like `pn['pore.coords'] += [0, 0, 5]`.

**Reply by likeduck** [reply]
Yes！I got it！Thank you！

**Comment by likeduck** [comment]
I also meet another problem.
I try to use ['pore.coords'] to define pore.left, like this（in 2D Voronoi network）: 
```
Ps = pn['pore.coords'][0] = 0 
pn.set_label(label='pore.left', pores=Ps)
```
but it is not very ideal. Could you tell me how can i define labels using coordinates? I can't them in openpnm website.

**Reply by jgostick** [reply]
You have a syntax error...should be `Ps = pn['pore.coords'][:, 0] == 0` to find all pores with an x-coordinate of 0, or if you want to find all pores with the same x-coord as the "minimum" x-coord then `Ps = pn['pore.coords'][:, 0] == pn['pore.coords'][:, 0].min()`.

**Reply by likeduck** [reply]
Thank You! It works!
